website_snippet_big_button/static/src/css/website_snippet_big_button.scss

```diff
@@ -0,0 +1,7 @@
+// Copyright 2015-2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+// Copyright 2019 Tecnativa - Cristina Martin R.
+// License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+.big-buttons a {
+    display: grid;
```

Comment:
If using the same styles as there were before in the sass file is enough, I'd prefer you to use that instead of grid, which is cool but seems like an overkill here.

Reply:
It's not enough with the previous version class, because when the link is modified the elements  are unified. In this way, each element is kept in a line. Maybe seems overkill here because I left text above and text below so that the user has 2 options, If they want can delete one of them.

Reply:
Isn't that because you changed the `<h3>` and `<p>` for 2 `<span>`?

Why did you need to change that? Does this mean that migrated websites buttons will no longer work?

If it was really needed, you can just do `.big-buttons a { span,i { display: block }}` instead.